src/components/Calculator.js

```diff
@@ -102,28 +141,36 @@ function OrangeBtn(props) {
         justifyContent: 'center',
         borderStyle: 'solid',
         borderWidth: 1,
-        borderColor: '#dfdfdf',
-        borderTop: 0,
+        boxSizing: 'border-box',
+        borderTopColor: first ? 'gray' : '#efefef',
+        borderTopWidth: first ? 1 : 0,
       }}
     >
       {string}
-    </div>
+    </button>
   );
 }
 
 OrangeBtn.defaultProps = {
-  string: 0,
+  string: '',
+  first: false,
+  clickCalc: () => {},
 };
 
 OrangeBtn.propTypes = {
   string: PropTypes.string,
+  first: PropTypes.bool,
+  clickCalc: PropTypes.func,
 };
 
 function GrayBtn(props) {
   const { string } = props;
   const { big } = props;
+  const { clickCalc } = props;
   return (
-    <div
+    <button
+      type="button"
+      onClick={() => { clickCalc(string); }}
```

Comment:
[ Optional Change ]

- [ ] What if you could create a file for this component and then import it into this Calculator component and pass props values to it so that this Calculator component remains lean and have less code?
Currently, this file has lots of code logic that can be moved to another file for their separate functions and then import and use here. That's the heartbeat of React, creating components and using them anywhere in your project. 👍🏽

Reply:
well resolved! ✔️